dev/Pager/InteractionTests/PagerTests.cs

```diff
@@ -529,6 +529,17 @@ public void ChangingDisplayModeTest()
 
                 SetNumberPanelDisplayMode();
                 VerifyNumberPanelDisplayMode();
+
+                ChangeNumberOfPages();
+                VerifyNumberOfPages("100");
```

Comment:
>100 [](start = 37, length = 3)

I would also verify the boundaries, since the display mode changes at 10 we should be testing 9 and 10 as well.

Reply:
Done